README.md

```diff
@@ -27,8 +27,8 @@ functionality to reduce the memory footprint and attack surface area of each
 microVM. This improves security, decreases the startup time, and increases
 hardware utilization. Firecracker has also been integrated in container
 runtimes, for example
-[Kata Containers](https://github.com/kata-containers/documentation/wiki/Initial-release-of-Kata-Containers-with-Firecracker-support)
-and [Weaveworks Ignite](https://github.com/weaveworks/ignite).
+[Kata Containers](https://github.com/kata-containers/kata-containers)
```

Comment:
Ok, now the markdown checker wants it to be:
```diff
-[Kata Containers](https://github.com/kata-containers/kata-containers)
-and [Flintlock](https://github.com/liquidmetal-dev/flintlock).
+[Kata Containers](https://github.com/kata-containers/kata-containers) and
+[Flintlock](https://github.com/liquidmetal-dev/flintlock).
```
Could you update that, please?

Reply:
I beat you to it already commits already in

Reply:
please squash the changes in a single commit (can use `git rebase -i` and `git push -f` -- no need for a new PR).

Reply:
done

Reply:
Please amend the message to remove these now-redundant lines:
```
doc: README.md Fixed formatting on the lines added

Fixed the formatting so the ci doesn't break anymore

Signed-off-by: Echo-Head-Wall <101990236+Echo-Head-Wall@users.noreply.github.com>
```

Reply:
fixed